app/options.ts

```diff
@@ -77,6 +82,23 @@ export const options: {
 			},
 		],
 	},
+	sortingOrder: {
+		since: '1.15.0',
+		category: 'Global',
+		type: 'choice',
+		default: SORTING_ORDER.DESCENDING,
+		description: 'Which sorting order to use',
```

Comment:
Would put the default in the description here too, so `...to use (default descending)`